src/remove_duplicate_single_line_statements.py

```diff
@@ -1,8 +0,0 @@
-import sys
```

Comment:
Why I removed this: didn't seem to break the build not to have it; and less is more.